src/endpoint.env

```diff
@@ -1,3 +1,3 @@
 LOCALSTACK_URL=http://localhost.localstack.cloud:4566
 OTLP_METRICS_URL=http://adot:5318/v1/metrics
-OTLP_TRACING_URL=http://jaeger:4318/v1/traces
\ No newline at end of file
+OTLP_TRACING_URL=http://adot:5318/v1/traces
```

Comment:
Does this change imply that Jeager will retrieve traces from the adot collector?

Reply:
yes, it is the `otlphttp `exporter in adot yaml configuration